service/datasource-service.py

```diff
@@ -144,84 +74,45 @@ def authenticate():
 def requires_auth(f):
     @wraps(f)
     def decorated(*args, **kwargs):
-        auth_method = get_var('auth') or "none"
+        auth_method = "none"
         if auth_method != "none":
             auth = request.authorization
             if not auth:
                 return authenticate()
             return f(*args, **kwargs)
         else:
             return f(*args, **kwargs)
-
     return decorated
 
 
-@app.route('/')
+@app.route("/get_excel", methods=["GET", "POST"])
 @requires_auth
 def get_entities():
-    file = get_var('file')
-    sheet = int(get_var('sheet') or 1) - 1
-    ids = get_var('ids') or "0"
-    ids = [int(x)-1 for x in ids.split(',')]
-    names = get_var('names') or "1"
-    names = [int(x)-1 for x in names.split(',')]
-    direction = get_var('direction') or "row"
-    start = get_var('start')
-    if not start and direction=="row":
-        start="1,2"
-    elif not start:
-        start = "2,1"
-    start = [int(x)-1 for x in start.split(',')]
-    logger.info("Get %s using request: %s" % (file, request.url))
-    since = request.args.get('since') or "0001-01-01"
-
+    logger.info("------Starting app-------")
```

Comment:
i think _get_var_ method should stay. It gives a nice flexibility and removing it will not encourage users to upgrade to the new version.